README.md

```diff
@@ -4,122 +4,145 @@ Kubernetes manifest files for [notification.canada.ca](https://notification.cana
+Notify components are deployed to Kubernetes using [helm](https://helm.sh/) and [helmfile](https://github.com/helmfile/helmfile). This allows us to do "diffs" on our environments, and apply applications surgically using labeles.
 
-- Environment variables
-- Target group bindings between the AWS network infrastructure and the Kubernetes cluster
-- Replica count patches (ex. How many pods of each type run in each environment)
+```bash
+helmfile -e dev -l app=notify-admin diff
+```
+
+```bash
+helmfile -e dev -l app=notify-admin sync
+```
 
 ## How are environment variables set?
 
-To set the variables for the API Lambda see [this](https://github.com/cds-snc/notification-terraform#awslambda-api)
+### Environment Variables
 
-`Kustomize` can dynamically inject environment variables when it compiles the configuration. To do this it reads out the environment variables and creates a `ConfigMap` object using an `.env` file that is in the same directory as the overlay that is being called from (ex. `/env/staging/.env`). As it is bad practice to save environment variables to a Git repository, the `.env` is ignored, and instead saved in an encrypted envelope using AWS KMS as `.env.enc.aws` files.
+Environment variables are set using a key/value pair in the values.yaml of the GC Notify helm charts, along with their overrides for each environment. [See the below section for adding a new environment variable.](##HowdoIaddanewenvironmentvariable?)
 
-This means that before the overlay is applied, the file needs to be decrypted.
+### Secrets
 
-### Decrypting environment variables
+Secrets are set using Kubernetes secrets, which ensures that we do not ever spit sensitive information out on helmfile diffs or logs. [To add a secret please follow this guide.](https://github.com/cds-snc/notification-terraform/blob/main/docs/creatingSecrets.md)
 
-You should leverage the appropriate commands in the Makefile:
 
-- the `make decrypt-staging` command that will decrypt environment variables in staging ;
-- the `make decrypt-production` command that will decrypt environment variables in production.
+## How do I add a new environment variable?
 
-```sh
-# You need to have the AWS credentials set up on your machine
-# under the profile name `notify-staging`.
-AWS_PROFILE=notify-staging make decrypt-staging
-# This creates a new decrypted file at env/staging/.env
-```
+### READ THIS
+
+If the value you are adding is a secret, DO NOT USE THIS METHOD. [Instead follow the instructions in the adding a secret document.](https://github.com/cds-snc/notification-terraform/blob/main/docs/creatingSecrets.md)
+
+### Steps for everything but API
 
-### Encrypting variables in staging/production
+To add an environment variable for admin or celery, you will need to update up to 3 files. 
```

Comment:
To add an environment variable for admin , API, celery, you will need to update up to 3 files.

Reply:
Changed